newIDE/app/src/Utils/Error.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import Paper from 'material-ui/Paper';
+
+/**
+ * Error component style.
+ */
+const styles = {
+    paper: {
```

Comment:
If there is so style needed, just remove this object and the whole styles object as well :)